api/admin/controller/sitewide_services.py

```diff
@@ -137,15 +137,61 @@ def process_delete(self, service_id):
             service_id, ExternalIntegration.SEARCH_GOAL
         )
 
-class StorageServicesController(SitewideServicesController):
-    def process_services(self):
-        detail = _("You tried to create a new storage service, but a storage service is already configured.")
-        return self._manage_sitewide_service(
-            ExternalIntegration.STORAGE_GOAL,
+class StorageServicesController(SettingsController):
```

Comment:
This should be moved to its own file or with other controllers but leaving it temporarily.